src/items/implementations.md

```diff
@@ -1,10 +1,64 @@
 # Implementations
 
-An _implementation_ is an item that can implement a [trait](items/traits.html) for a
-specific type.
+An _implementation_ is an item that associates items with an *implementing type*.
+
+There are two types of implementations: Bare implementations and [trait] 
+implementations.
 
 Implementations are defined with the keyword `impl`.
 
+## Bare Implementations
```

Comment:
Aren't these called "inherent implementations"?

Reply:
I've seen them called both. Inherent is probably a better name though. Will fix tonight.